data/supportserver/aytests/aytests.cgi

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# do some variable expansion in autoyast profiles during download
+FILE=${REQUEST_URI#/aytests/}
+[ ! -f "/srv/www/htdocs/aytests/$FILE" ] && exit 1
+
+# use real VM's MAC addres to test udev rules
+MAC=$( echo $( arp | grep "$REMOTE_ADDR" ) | cut -d ' ' -f 3 )
+[ -z "$MAC" ] && exit 1
+echo "replacing MAC '$MAC' in '$FILE'" > /dev/stderr
+
+echo "Content-type: text/xml"
+echo
+sed -e "s|{{MAC1}}|$MAC|g" \
+    -e "s|{{REPO1_URL}}|http://10.0.2.1/aytests/files/repos/sles12|g" \
```

Comment:
This is IP address of support server. It depends on openQA network infrastructure and is unlikely to be changed. It might be put into some variable, but it is for another pull request as it already is on several locations in the code.